src/app/modules/item/components/item-children-edit/item-children.ts

```diff
@@ -1,6 +1,8 @@
+import { ItemType } from '../../../../shared/helpers/item-type';
 
 export interface ItemChild {
-  id: string,
+  id?: string,
   title: string | null,
   order: number,
+  type?: ItemType,
```

Comment:
It is not optional in item so it makes probably sense to have it non-option here and get it from the service (may be useful to display an icon is front anyway)

Reply:
I was thinking about doing it in its own PR, But it's done, I've just moved it to this PR, tell me if you prefer that I display icons instead of what as been done in the design

Reply:
Ah I didn't mean that the icon has to be done in this PR but that including the type as non optional would allow icons later. Ok I'll check what you have done.

Reply:
What you have done is ok.